src/models/relations.ts

```diff
@@ -122,20 +127,6 @@ export class Relations extends TypedEventEmitter<RelationsEvent, EventHandlerMap
             return;
         }
 
-        const relation = event.getRelation();
```

Comment:
Events can only be added to `this.relations` if they pass the identical checks in `addEvent`, so these checks are superfluous.